multihost_test/bz_automation/test_pam_userdb.py

```diff
@@ -0,0 +1,136 @@
+"""
+This script contains tests for pam_userdb database migration.
+"""
+
+import os
+import re
+import pytest
+
+db_file = "test.gdbm"
+
+
+def client_version(multihost):
+    if int(re.findall(r'\d+', multihost.client[0].distro)[0]) >= 10:
+        return True
+
+
+@pytest.mark.tier1
+class TestPamBz(object):
+    def test_gdbm_dump_gdbm_load(self, multihost):
```

Comment:
This test case checks gdbm utilities internals and we aren't in charge of that, so remove it

Reply:
removed